File: web/firebase/fireapp/models/route.py
from django.db import models

class Route(models.Model):
    Id = models.CharField(max_length=200)
    Destiny = models.CharField(max_length=100)
    Origin = models.CharField(max_length=100)
    DestinyName = models.CharField(max_length=100)
    OriginName = models.CharField(max_length=100)
    Number = models.IntegerField()

File: driver/lib/models/route.dart
class RouteModel {

  final String id;
  final String destiny;
  final int number;
  final String origin;
  final num? price;

  RouteModel ({
    required this.id,
    required this.destiny,
    required this.number,
    required this.origin,
    this.price,
  });
  
}